Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -410,15 +410,15 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
  * @parameter range attributed string range to use.
  * @parameter frameSize size parameters of the frame to fit the text into.
  *
- * @return _CTFrame* created using the given parameters
+ * @return CTFrameRef created using the given parameters
  */
-static _CTFrame* _DWriteGetFrame(CFAttributedStringRef string, CFRange range, CGRect frameSize) {
+static CTFrameRef _DWriteGetFrame(CFAttributedStringRef string, CFRange range, CGRect frameSize) {
     RETURN_NULL_IF(!string);
 
-    _CTFrame* frame = [[_CTFrame new] autorelease];
-    if (range.length <= 0) {
-        return frame;
-    }
+    CTFrameRef frame = __CTFrame::CreateInstance(kCFAllocatorDefault);
```

Comment:
This is the only one occurrence. 

---
In reply to: [110012166](https://github.com/Microsoft/WinObjC/pull/2427#discussion_r110012166) [](ancestors = 110012166)